### ISSUE TITLE
使用多GPU进行训练时报出如下错误，单GPU没问题

### ISSUE THREAD

**Issue by DendiHust** [bojone/bert4keras#116]
Exception: Layers should have equal number of output tensors and output masks. Layer layer_normalization_1 has 1 output tensors and 2 output masks.

**Comment by bojone** [comment]
多GPU没经验，抱歉～

**Comment by yiyangiliu** [comment]
你试试keras自带的[multi_gpu_model](https://keras.io/utils/#multi_gpu_model)，建立一个多gpu模型

**Comment by bojone** [comment]
https://github.com/bojone/bert4keras/blob/master/examples/task_seq2seq_autotitle_multigpu.py
